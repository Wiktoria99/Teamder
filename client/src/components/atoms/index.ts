export * from './BackButton';
export * from './CustomTextField';
export * from './CustomButton';
export * from './SideBarButton';
