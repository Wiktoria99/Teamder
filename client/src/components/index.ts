export * from "./atoms";
export * from "./atoms";
export * from "./molecules";
export * from "./organisms";
