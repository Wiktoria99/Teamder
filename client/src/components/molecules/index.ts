export * from './RegisterSteps';
export * from './TeamItem';
export * from './Suggestions';
export * from './Sidebar';
export * from './TeamList';
export * from './SuggestionItem';
export * from './SuggestionList';
export * from './NotificationItem';
export * from './NotificationList';
