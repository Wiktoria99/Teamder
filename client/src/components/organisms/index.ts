export * from './Layout';
export * from './FormWrapper';
export * from './MainWrapper';
export * from './BarWrapper';
