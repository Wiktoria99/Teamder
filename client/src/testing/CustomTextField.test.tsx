import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { cleanup } from '@testing-library/react';
import { shallow} from "enzyme";
import { CustomTextField } from '@/components';

afterEach(cleanup);

it('CustomTextField renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<CustomTextField />, div);
});

it('CustomText renders correctly', () => {
  const wrapper = shallow(<CustomTextField value="testing" />);
  expect(wrapper.props().value).toEqual('testing');
});

it('CustomButton matches snapshot', () => {
  const tree = renderer.create(<CustomTextField />).toJSON();
  expect(tree).toMatchSnapshot();
});
