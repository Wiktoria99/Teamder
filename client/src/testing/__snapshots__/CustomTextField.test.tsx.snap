// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton matches snapshot 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-1"
>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
